### PR TITLE
Fix migrate typos.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: staging/migrate.sql [hunk 1]
```diff
@@ -51,7 +51,7 @@ TRUNCATE TABLE staging_asmt;
 TRUNCATE TABLE staging_asmt_score;
 TRUNCATE TABLE staging_item;
 
-TRUNCATE TABLE staging_acommodation_translation;
```
Comment (agorina) [inline staging/migrate.sql]: This was a typo.

File: staging/migrate.sql [hunk 2]
```diff
@@ -646,22 +646,22 @@ INSERT INTO reporting.target ( id, claim_id, code, description)
   WHERE rt.id IS NULL;
 
 -- ------------ Common Core Standard -------------------------------------------------------------------------
-UPDATE reporting.common_core_standard rc
-    JOIN staging.staging_common_core_standard sccs ON sccs.id = rccs.id
- SET
-    rccs.subject_id = sccs.subject_id,
-    rccs.natural_id = sccs.natural_id,
-    rccs.description = sccs.description;
+UPDATE reporting.common_core_standard rccs
```
Comment (agorina) [inline staging/migrate.sql]: And this was wrong.

File: staging/migrate.sql (hunk 1 above)
Comment (chdietz) [inline staging/migrate.sql]: spelling error, why didn't it fail ?

Comment (agorina) [inline staging/migrate.sql]: It fails when I run this script. I am assuming that this is just a typo in here, not in the migrate code?

**Pull request merged**